packages/suite/src/middlewares/suite/eventsMiddleware.ts

```diff
@@ -67,6 +67,10 @@ const eventsMiddleware =
             }
         }
 
+        if (action.type === DEVICE.FIRMWARE_VERSION_CHANGED) {
+            // TODO: show warning? move this to different middleware?
```

Comment:
TODO log to Sentry (I can do it).

Reply:
[Example log in Sentry
](https://satoshilabs.sentry.io/issues/6270069178/?project=5193825&query=release%3A25.2.0.web.e4fa886a85ad6bb49b08e491a265cbf67bb0db12&referrer=release-issue-stream)Mocked by changing [this line](https://github.com/trezor/trezor-suite/blob/5b7ca32b9abf7c35f43b10e8b35c6bbfa9c293c6/packages/connect/src/device/Device.ts#L1004).